### DISCUSSION TITLE
Relaxation in lmp

### DISCUSSION THREAD

**Discussion by alib-l** [deepmodeling/deepmd-kit#758] [Q&A]
Hello,
I have trained a model for SiO2 and frozen it (the loss function converged and the predicted values were accurate). Then I run four lmp simulations to relax it with "fix NVT", but I got different results each time. In 3 of them, the structure was destroyed and the atoms dispersed, but in one of them, the cell was relaxed. Could you please help me with this issue? 
Thanks,

**Comment by njzjz** [comment]
> but I got different results each time.

Can you post the lammps command you used?

Also, you may need to add more data into your training set.
